server/app/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "vad_mode":3,
+    "silence_ratio":0.9,
+    "lowest_dBFS":-18,
+    "shortest_length_for_sentence":0.15,
```

Comment:
Why is the shortest length 0.15? I think Aanchan suggested with 0.3s

Reply:
fixed